TestHelloWorld.java

```diff
@@ -0,0 +1,30 @@
+import static org.junit.Assert.assertEquals;
+
+import java.io.ByteArrayOutputStream;
+import java.io.InputStream;
+import java.io.PrintStream;
+import org.junit.Test;
+public class TestHelloWorld{
+	
+	@Test
+	public void testHelloWorldString(){
+		//save the system.in and system.out

```

Comment:
Lines 11-17 are "before the test" setup types of activities. JUnit provides a nice mechanism for these setup and teardown types of things.